src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.csproj

```diff
@@ -25,7 +25,65 @@
     <Folder Include="Properties\" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Update="ProjectSystem\VS\PropertyPages\DebugPageControl.xaml.cs">
+    <Compile Include="GlobalSuppressions.cs" />
```

Comment:
Yes we switched the projects to dogfood the SDK only in the master which is now 15.3. We didn't update the 15.1\15.2 branches.